tools/generator/rules.py

```diff
@@ -21,14 +25,15 @@ def make_rules(game, out_dir: Path) -> None:
         'game_state.hh'
     ]
     for tpl in base_files:
-        gen.template(tpl)
+        gen.template(os.path.join('src', tpl))
 
     for action in game["function"]:
         if not action.get("fct_action"):
             continue
-        gen.template('action_template.hh',
-                     out_name='action_{}.hh'.format(action['fct_name']),
-                     action=action)
-        gen.template('action_template.cc',
-                     out_name='action_{}.cc'.format(action['fct_name']),
-                     action=action)
+        for ext in ('hh', 'cc'):
+            gen.template(os.path.join('src', 'action_template.' + ext),
```

Comment:
nit: `'src/action_template.{}'.format(ext)` for consistency

Reply:
done